src/components/tracked-controls.js

```diff
@@ -84,7 +85,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     var controllerQuaternion = this.controllerQuaternion;
     var controllerEuler = this.controllerEuler;
     var deltaControllerPosition = this.deltaControllerPosition;
-    var hand = (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
+    var hand = this.data.hand || (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
```

Comment:
is the intent here to allow the arm model to use something different than gamepad hand property when present?  or to allow specifying default hand to use when gamepad doesn't provide?  if the latter, I had actually renamed it to defaultHand before @dmarcos asked to remove it entirely and just use right DEFAULT_HANDEDNESS constant. so I am not sure we want this, per that prior discussion?